AWS Output's don't work with IAM Role Service Accounts.

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks! -->

```falcosidekick-7cd6dc784b-zxbph falcosidekick 	status code: 400, request id: 25768792-8667-4b8c-a281-049320f4ab36

falcosidekick-7cd6dc784b-zxbph falcosidekick 2021/04/07 17:27:05 [INFO]  : AWS CloudWatchLogs - Log Stream not configured creating one called falcosidekick-logstream

falcosidekick-7cd6dc784b-zxbph falcosidekick 2021/04/07 17:27:05 [ERROR] : AWS CloudWatchLogs - AccessDeniedException: User: arn:aws:sts::164382793440:assumed-role/falco-prow-test-infra2021031522165656100000000e/i-0c86c20993deb6679 is not authorized to perform: logs:CreateLogStream on resource: arn:aws:logs:us-west-2:164382793440:log-group:k8s-meetup-falco-alerts:log-stream:falcosidekick-logstream
falcosidekick-7cd6dc784b-zxbph falcosidekick 	status code: 400, request id: f77c160b-9078-452b-9139-b62045e7ecaf
```

**Describe the bug**

<!-- A clear and concise description of what the bug is. -->

FalcoSidekick's AWS Cloudwatch output doesn't work with IRSA (IAM Role Service Accounts). 

**How to reproduce it**
```
1. Install Falcosidekick
helm install falcosidekick \
  --set webui.enabled=true \
  --set webui.image.tag="v1.0.0" \
  --set config.aws.cloudwatchlogs.loggroup="k8s-meetup-falco-alerts" \
  --set config.aws.region="us-west-2" \
  falcosecurity/falcosidekick ```

2. Setup IAM role and OIDC annotation `{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "logs:CreateLogGroup",
                "logs:CreateLogStream",
                "logs:PutLogEvents",
                "logs:DescribeLogStreams"
            ],
            "Resource": [
                "arn:aws:logs:*:*:*"
            ]
        }
    ]
}

Assume-role policy.
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Sid": "",
      "Effect": "Allow",
      "Principal": {
        "Federated": "arn:aws:iam::1234567890:oidc-provider/oidc.eks.us-west-2.amazonaws.com/id/DDE52ABC393DFEED92ECB2AB8FB56F7B"
      },
      "Action": "sts:AssumeRoleWithWebIdentity",
      "Condition": {
        "StringEquals": {
          "oidc.eks.us-west-2.amazonaws.com/id/DDE52ABC393DFEED92ECB2AB8FB56F7B:sub": "system:serviceaccount:prow-monitoring:falcosidekick"
        }
      }
    }
  ]
}

3. Annotate SA `kubectl annotate serviceaccount -n prow-monitoring falcosidekick \
eks.amazonaws.com/role-arn=arn:aws:iam::164382793440:role/falco-prow-test-infra-falcosidekick`

4. Generate event's and see if falcosidekick is able to use IRSA.

```


<!-- Minimal and precise steps to reproduce the bug. -->

**Expected behaviour**

It would pull the role credentials from the SA not from the node role.

**Version

Falco sidekick helm chart install.

--- a/outputs/aws.go
+++ b/outputs/aws.go
@@ -46,7 +46,7 @@ func NewAWSClient(config *types.Configuration, stats *types.Statistics, promStat
 		return nil, errors.New("Error while creating AWS Session")
 	}
 
-	_, err = sts.New(session.New()).GetCallerIdentity(&sts.GetCallerIdentityInput{})
+	_, err = sts.New(sess).GetCallerIdentity(&sts.GetCallerIdentityInput{})
 	if err != nil {
 		log.Printf("[ERROR] : AWS - %v\n", "Error while getting AWS Token")
 		return nil, errors.New("Error while getting AWS Token")

